x/ibc/applications/transfer/types/keys.go

```diff
@@ -30,6 +29,10 @@ const (
 
 	// DenomPrefix is the prefix used for internal SDK coin representation.
 	DenomPrefix = "ibc"
+
+	// EscrowPrefix is the prefix used for IBC escrow addresses to avoid address collisions
+	// with Public Key addresses.
+	EscrowPrefix = "ibc"
```

Comment:
Shouldn't this be longer? The more characters there are here the harder it is to find a collision I believe?

I might suggest something like: `ibc-transfer-escrow-prefix`